scss/6-components/_targeted-content.scss

```diff
@@ -88,9 +88,6 @@
   .cads-targeted-content__expand-all {
     display: inline-block;
 
-    @extend .cads-button;
```

Comment:
Noted this in the commit but this caused problems with refactoring. Extend is tricksy and best avoided, see https://csswizardry.com/2014/11/when-to-use-extend-when-to-use-a-mixin/. Simple fix here was to explicitly add the required classes to the expand button in the JS.